src/main/java/com/sollace/coppergolem/client/CopperGolemEntityModel.java

```diff
@@ -36,32 +35,32 @@ public CopperGolemEntityModel(ModelPart root) {
         this.leftArm = body.getChild(EntityModelPartNames.LEFT_ARM);
         this.rightLeg = root.getChild(EntityModelPartNames.RIGHT_LEG);
         this.leftLeg = root.getChild(EntityModelPartNames.LEFT_LEG);
-     }
+    }
 
     public static TexturedModelData getTexturedModelData() {
         ModelData data = new ModelData();
         ModelPartData root = data.getRoot();
 
         ModelPartData body = root.addChild(EntityModelPartNames.BODY, ModelPartBuilder.create()
-                .uv(0, 13).cuboid(-4, -7, -3, 8, 7, 5), ModelTransform.NONE);
+            .uv(0, 13).cuboid(-4, -7, -3, 8, 7, 5), ModelTransform.NONE);
 
         body.addChild(EntityModelPartNames.HEAD, ModelPartBuilder.create()
-                        .uv(36, 31).cuboid(-1, -1, -1.5F, 2, 2, 2)
-                        .uv(0, 0).cuboid(-4, -5, -4, 8, 5, 7)
-                        .uv(26, 16).cuboid(-1, -7, -1.5F, 2, 2, 2)
-                        .uv(0, 32).cuboid(-1.5F, -10, -2, 3, 3, 3), ModelTransform.pivot(0, -6, 0))
-                .addChild(EntityModelPartNames.NOSE, ModelPartBuilder.create()
-                        .uv(36, 26).cuboid(-1.5F, -1, 0, 2, 3, 2), ModelTransform.pivot(0.5F, -1, 3));
+                .uv(36, 31).cuboid(-1, -1, -1.5F, 2, 2, 2)
+                .uv(0, 0).cuboid(-4, -5, -4, 8, 5, 7)
+                .uv(26, 16).cuboid(-1, -7, -1.5F, 2, 2, 2)
+                .uv(0, 32).cuboid(-1.5F, -10, -2, 3, 3, 3), ModelTransform.pivot(0, -6, 0))
```

Comment:
See previous note about wrapped method chains. Some of these I've tried to line them up with the start of the first method in the chain as well.